experiments/interpretation/dutchf3_patch/local/train.py

```diff
@@ -90,13 +91,23 @@ def run(*options, cfg=None, debug=False):
                                       default.py
         cfg (str, optional): Location of config file to load. Defaults to None.
         debug (bool): Places scripts in debug/test mode and only executes a few iterations
+        input (str, optional): Location of data if Azure ML run
```

Comment:
It's still possible to run the script without AML, isn't? Should the comments mentioned what the params should be for local run?

Reply:
Good idea. Updated here: https://github.com/microsoft/seismic-deeplearning/pull/186/commits/8e0c2d700dd5b189d150b94ea76416f61199be8b